src/Validation/Validation.php

```diff
@@ -1098,6 +1098,68 @@ public static function longitude($value, array $options = [])
     }
 
     /**
+     * Check that the input value is within the ascii byte range.
+     *
+     * This method will reject all non-string values.
+     *
+     * @param string $value The value to check
+     * @return bool
+     */
+    public static function ascii($value)
```

Comment:
Would it be too much for 3.1 to change all to isX() and deprecate the other ones in 3.x for 4.0?